src/templates/export-map.js

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+export const exportMapToHTML = options => {
+  return `
+    <!DOCTYPE html>
+    <html>
+    <head>
+      <meta charset="UTF-8"/>
+      <title>Kepler.gl embedded map</title>
+    
+      <!-- Load React/Redux -->
+      <script src="https://unpkg.com/react@15.6.2/dist/react.min.js" crossorigin></script>
+      <script src="https://unpkg.com/react-dom@15.6.2/dist/react-dom.min.js" crossorigin></script>
+      <script src="https://unpkg.com/redux@3.7.2/dist/redux.js" crossorigin></script>
+      <script src="https://unpkg.com/react-redux@4.4.9/dist/react-redux.min.js" crossorigin></script>
+    
+      <!-- Load Kepler.gl latest version-->
+      <script src="https://unpkg.com/kepler.gl/umd/keplergl.min.js"></script>
+      <!-- You can also specify a specific versions by doing the following -->
+      <!--<script src="https://unpkg.com/kepler.gl@0.2.2/umd/keplergl.min.js"></script>-->
+    
+      <!--MapBox token-->
+      <script>
+        /**
+         * Provide your MapBox Token
+         * This will be used in app.js (imported later)
+         * */
+        const MAPBOX_TOKEN = '${options.mapboxApiAccessToken || 'PROVIDE_MAPBOX_TOKEN'}';
```

Comment:
Talked to Chris about this, maybe we should create a unique token for exported html maps, and rotate this token every month. This way we have a better way to tracking how many map views are from exported html maps.

Reply:
For tracking, you may also want to include a version number with the approach in #56 .

Reply:
> For tracking, you may also want to include a version number with the approach in #56 .
> 
> When we do rotate token, would that break their exported HTML? We may want to inform the users somehow that we've included a default token but expect them to change it if they need it to work as long as they need it to (via their own token). It would be great if users can swap it with their own respective token if they needed to.

@chrisirhc 
We currently allow users to provide their own Mapbox Token as they export the map. I can update the interaction and provide a more suitable message for this new use case.

When you mention "it would be great if users can swap it with their own....", are you referring at any given time after they download the html file? or just when they initially export their maps?

Reply:
> When you mention "it would be great if users can swap it with their own....", are you referring at any given time after they download the html file? or just when they initially export their maps?

Consider thinking from the user's point of view: Since the Mapbox token rotates, the user would need to take certain actions if the Mapbox token we provided expired, after the html is exported.
* Would it be obvious to them what action they need to take? (Does the user see an error? Does the error give them information on how to resolve the issue?)
* What steps are required by the user to resolve the issue?
* Are those steps difficult?

Reply:
Got it.

Currently in Kepler.gl we simply show a log error message: "Mapbox Token is required but nothing in the UI".

Before we implement checks and suggestions in this feature, we should focus on kepler.gl framework messaging by implementing the following:
- Show a notification error if Mapbox Token is missing
- As part of the message we can include a permalink to the user guide where we explain how to update the token when it is expired.

The above changes will give us more flexibility in providing steps and solutions it will also improve the general kepler.gl experience.

As part of the "Export map experience":
- We can always add a default mapbox token with the ability to override it when they export the map.
- When we rotate the token, the mapbox error notification will provide the link with instructions on how to update their token

Reply:
Sounds good to me. Thanks 👍

Reply:
I created a new issue to track the necessary work to add an error notification when mapbox token is missing: https://github.com/uber/kepler.gl/issues/426